recipes/socat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "socat" %}
+{% set version = "1.7.3.2" %}
+{% set md5 = "607a24c15bd2cb54e9328bfbbd3a1ae9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: http://www.dest-unreach.org/{{ name }}/download/{{ name }}-{{ version }}.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/
```

Comment:
FWICT, `lib/` [is the default for `rpaths`](https://conda.io/docs/building/meta-yaml.html#rpaths). You can *probably* get rid of lines 17-18. I haven't used this feature of conda-build before, so I could be wrong about this.